tests/pages/Favoris.page.test.tsx

```diff
@@ -0,0 +1,150 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { GetServerSidePropsResult } from 'next'
+import { GetServerSidePropsContext } from 'next/types'
+import React from 'react'
+
+import { uneListeDeRecherches, uneListeDOffres } from 'fixtures/favoris'
+import { mockedFavorisService } from 'fixtures/services'
+import Favoris, {
+  getServerSideProps,
+} from 'pages/mes-jeunes/[jeune_id]/favoris'
+import { FavorisService } from 'services/favoris.service'
+import { withMandatorySessionOrRedirect } from 'utils/auth/withMandatorySessionOrRedirect'
+import { ApiError } from 'utils/httpClient'
+import withDependance from 'utils/injectionDependances/withDependance'
+
+jest.mock('utils/auth/withMandatorySessionOrRedirect')
+jest.mock('utils/injectionDependances/withDependance')
+
+describe('Favoris', () => {
+  const offres = uneListeDOffres()
+  const recherches = uneListeDeRecherches()
+
+  describe('client side', () => {
+    beforeEach(async () => {
+      render(<Favoris offres={offres} recherches={recherches} pageTitle={''} />)
+    })
+
+    it('affiche la liste de ses offres', () => {
```

Comment:
```suggestion
    it('affiche la liste des offres du jeune', () => {
```
nitpick: ses? à qui?

Reply:
A oui, au début j'avais mis un describe `pour un jeune donné`, mais @arthurlbrjc m'a dit de l'enlever à juste titre. Mais je n'ai pas mis à jour le nom du test…